test/test_unit.py

```diff
@@ -242,12 +251,12 @@ def testSetPadding(self):
     def testSetFrameSet(self):
         seq = FileSequence("/cheech/chong.1-5#.exr")
         seq.setFrameSet(FrameSet("10-20"))
-        self.assertEquals("/cheech/chong.10-20#.exr", str(seq))
+        self.assertEquals(u"/cheech/chong.10-20#.exr", six.text_type(seq))
```

Comment:
Looks like all the `assertEquals` calls generate a `DeprecationWarning` now in py3 so we will need to swap them to `assertEqual`